docs/README.md

```diff
@@ -254,6 +254,24 @@ Metrics provided
 | :------------------------------------ | :----------------------- |
 | alfresco.status.readonly              |                          |
 
+## License metrics
+The license metrics bindings will provide metrics about Alfresco license. Enterprise-only feature.
+
+**Control Property**: `alfred.telemetry.binder.license.enabled`
+
+Metrics provided
+
+| Name                                  | Available tags           |
+| :------------------------------------ | :----------------------- |
+| license.valid                         |                          |
+| license.maxUsers                      |                          |
```

Comment:
Following on my previous comment: 
* `license.users` with the tag `max` keeps room for addition of e.g. a tag `active`. 

Same for other metrics:
* `license.docs` tags: `max`
* `license.cluster.enabled`
* ...